BUILD.bazel

```diff
@@ -1,17 +1,25 @@
-# Marker file indicating this folder is a Bazel package.
-# Needed so that tsconfig.json can be referenced from BUILD rules.
+package(default_visibility = ["//visibility:public"])
 
+# Needed so that tsconfig.json can be referenced from BUILD rules.
 exports_files(["tsconfig.json"])
 
-# Temporary target to allow `bazel test ...`
-# TODO(alexeagle): remove as soon as there is any other test in the repo
-genrule(
-    name = "true",
-    outs = ["true.sh"],
-    cmd = "echo true > $@",
-)
-
-sh_test(
-    name = "tautology_test",
-    srcs = [":true.sh"],
+filegroup(
```

Comment:
It isn't clear to me if I should be using the dependencies installed from running Yarn at the root for tests or if I should do something similar to the `ngrx_compile_time_deps` for tests.

Reply:
You have both options.
In the case of tests, your users probably don't expect to build/run these from source. At the same time, your users *would* download and install the `ngrx_compiletime_deps` so it's not nice to add test-only dependencies there.
I think I'd probably re-use the `node_modules` directory installed by yarn (note you can use `bazel run @yarn//:yarn` to do that with Bazel's version of node and yarn).

Reply:
Good deal. This PR follows your suggestion and re-uses the top level `node_modules` directory.